src/components/MaterialUI/DataDisplay/PredefinedElements/ActionModal.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Modal from "./../Modal";
+import ModalProps from "./../modalProps";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import sharedMessages from "../../../../sharedMessages";
+import { FormattedMessage } from "react-intl";
+
+const useStyles = makeStyles(theme => ({
+	actionPanel: {
+		float: "right",
+	},
+	message: {
+		maxWidth: "480px",
+		wordWrap: "normal",
+	},
+}));
+
+const ActionModal = ({
+	title,
+	message,
+	open,
+	actions, // Array of objects containing three properties: label, handler, isPrimary
+	backdropClickCallback,
+}) => {
+	const classes = useStyles();
+
+	const modalProps = new ModalProps();
+
+	const titleComponent = title || <FormattedMessage {...sharedMessages.confirmation} />;
+	const messageComponent = (
+		<div className={classes.message}>
```

Comment:
If it is only the text that is passed, this can be skipped and pass the variable directly in the modal